lava_test_plans/qemu.jinja2

```diff
@@ -112,6 +115,18 @@
             compression: {{OVERLAY_MODULES_URL_COMP}}
 {% endif %}
 {% endif %}
+{# MMTest-specific overlay settings  #}
+{% if OVERLAY_MMTESTS_URL is defined %}
+          mmtests:
+            url: {{ OVERLAY_MMTESTS_URL }}
+            path: /mmtests
```

Comment:
if we can make this a variable to be overridded it would be good.
like the KSELFTEST_PATH, please look how that is used.

Reply:
Good idea. Reworked.